uwhvz/settings/production.py

```diff
@@ -44,3 +44,22 @@
 # SESSION_COOKIE_SECURE = True
 # SECURE_SSL_REDIRECT = True
 # SECURE_HSTS_INCLUDE_SUBDOMAINS = True
+
+LOGGING = {
```

Comment:
This seems pretty cool! @tso what's your opinion on this? Unsure if we need the server error emails now?